src/server/api/controllers/boards.controller.js

```diff
@@ -59,7 +59,15 @@ const getBoardsByCreatorId = async (id) => {
 
   const boards = await knex('users')
     .join('boards', 'users.id', '=', 'boards.creatorId')
-    .select('*')
+    .select(
+      'users.fullName',
```

Comment:
The frontend already knows about the username, you do not need to return it here.